packages/docs/concepts/events-and-actions.yaml

```diff
@@ -194,7 +178,7 @@ _ref:
 
             The `onMountAsync` event is triggered every time a block is mounted, but does not render the block in loading.
 
-            # Action types
+            ## Action types
 
             The following actions can be used:
 
```

Comment:
`Message` was renamed to `DisplayMessage`

 The following need to be removed:
JsAction
Notification

The following actions need to be added:
Login
Logout
Fetch
ResetValidation
Break
Wait

Please update these links, and then update the individual action docs so the links match up. I am happy if you leave TODOs for the actions that don't have docs.

For Break @Gervwyk can update the Break action docs as part of his changes.